packages/devtools_app/lib/src/debugger/flutter/debugger_controller.dart

```diff
@@ -77,6 +77,15 @@ class DebuggerController extends DisposableController
 
   ValueListenable<String> get exceptionPauseMode => _exceptionPauseMode;
 
+  final ValueNotifier<bool> _librariesVisible = ValueNotifier(false);
+
+  ValueNotifier<bool> get librariesVisible => _librariesVisible;
```

Comment:
Nit: make the LHS a ValueListenable to make writes more difficult.

Reply:
oops - removed; this was for a separate PR